streams/src/test/java/org/apache/kafka/streams/integration/ErrorHandlingIntegrationTest.java

```diff
@@ -52,36 +52,46 @@
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+@Timeout(600)
 @Category(IntegrationTest.class)
 public class ErrorHandlingIntegrationTest {
 
+    private final String testId;
+    private final String appId;
+    private final Properties properties;
+
+    // Task 0
+    private final String inputTopic;
+    private final String outputTopic;
+    // Task 1
+    private final String errorInputTopic;
+    private final String errorOutputTopic;
+
     private static final EmbeddedKafkaCluster CLUSTER = new EmbeddedKafkaCluster(1);
 
-    @BeforeClass
+    ErrorHandlingIntegrationTest(final TestInfo testInfo) {
+        testId = safeUniqueTestName(getClass(), testInfo);
+        appId = "appId_" + testId;
+        properties = props();
+
+        inputTopic = "input" + testId;
+        outputTopic = "output" + testId;
+
+        errorInputTopic = "error-input" + testId;
+        errorOutputTopic = "error-output" + testId;
+    }
```

Comment:
I cannot remember, so maybe there was no reason. When I was rebasing I noticed that this test has since been deleted (or moved to `org.apache.kafka.connect.integration`) so I believe it is safe to say I do not need to address this?